packages/browser-agent-core/common/wrap/index.js

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
What does this file do?

Reply:
remnant of Martin's work, but serves as a single export file (imports a lot of different files, and exports them all, for ease of use)